### PR TITLE
Fixes #22518: Recent changes drop down cannot be read

### PR DESCRIPTION
**Pull request by clarktsiory**
https://issues.rudder.io/issues/22518

I removed the seconds from the formatted date (see the unit tests for examples of what will be displayed)

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/elm/rules/tests/ViewRepairedReportsTest.elm [hunk 1]
```diff
@@ -0,0 +1,42 @@
+module ViewRepairedReportsTest exposing (..)
```
Comment (amousset) [inline webapp/sources/rudder/rudder-web/src/main/elm/rules/tests/ViewRepairedReportsTest.elm:1]: running this test in CI requires changes in a pom.xml somewhere (the one that calls elm-test)

**Comment by VinceMacBuche** [comment]
LGTM, apart Alexis' remark I'll accept as soon as the change in pom.xml is done

**Comment by clarktsiory** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-web/src/main/elm/rules/tests/ViewRepairedReportsTest.elm (hunk 1 above)
Comment (clarktsiory) [reply webapp/sources/rudder/rudder-web/src/main/elm/rules/tests/ViewRepairedReportsTest.elm:1]: Good catch ! Added it in https://github.com/Normation/rudder/pull/5134/commits/92292d513be798047be84c06be9a6ea0b5b5edf5

**Review by amousset — APPROVED** [review]

**Comment by VinceMacBuche** [comment]
OK, squash merging this PR

**Pull request merged**